packages/server/api/routes/reviews.router.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const reviewsController = require('../controllers/reviews.controller');
+
+router.get('/rating/latest', (req, res) => {
```

Comment:
Instead of naming endpoint as '/rating/latest' I would suggest that you keep endpoint as /reviews and pass parameters as sortBy=latest-rated and a 'limit' parameter

Reply:
Thank you so much for your feedback! 
I have fixed the typo and for sortBy with parameter, we are all agree in class that we will keep the code as it is 
since for now and future, we might not need to pass parameters